blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md

```diff
@@ -0,0 +1,300 @@
+
+**Define compliance rules:** These compliance rules are encoded in the smart contract, allowing for faster processing and lower protocol fees that can scale as demand and regulatory complexity grow. It provides flexibility for issuers to set rules on who can hold the token, how many investors can hold the token, and how it can be transferred.
+
+**Store, manage, and distribute the token:** Tokens are stored in a software "vault" and supported by a custody service to manage digital twins (i.e., to maintain the link between an existing asset and its digital token representative). Tokenization service providers offer issuers an automated and reportable way to manage token distribution as well as future distribution events.
+
+**Execute corporate acts:** Allows the issuer to input some details and create a corporate action, for which the mechanism determines rights, schedules communications, distributes capital (if necessary), and updates records. Tokenization significantly reduces the costs of these corporate actions.
+  
+
+## The life cycle of an asset
+
+The life cycle of an asset that has been tokenized on the blockchain typically includes the following stages:
+
+**Creation:** The asset is tokenized by creating a smart contract on the blockchain that represents the asset and defines its ownership and transfer rules. A unique token is issued for the asset and stored in the smart contract.
+
+**Verification:** The ownership and authenticity of the asset are verified through the use of documents, certificates, or other forms of evidence.
+
+**Trading:** The asset is listed for sale on a marketplace or exchange that supports the trading of tokenized assets. Potential buyers can purchase the asset using the token that represents it.
+
+**Transfer:** When the asset is sold, the token representing the asset is transferred from the seller's wallet to the buyer's wallet. This transfer is recorded on the blockchain and is secured by the smart contract that was created during the creation stage.
+
+**Settlement:** After the transfer is complete, the asset is considered to be in the possession of the buyer. The settlement process may involve the transfer of physical ownership of the asset, or it may simply involve the transfer of ownership rights as represented by the token.
+
+**Redeemed:** In some cases, an asset may be redeemed by the original owner or by a third party. This process typically involves the transfer of the token back to the original owner or to the third party, and may be accompanied by the transfer of physical ownership or ownership rights.
```

Comment:
**Redeemed:** In some cases, an asset may be redeemed by the original owner or by a third party. This process typically involves the transfer of the token back to the original owner or the third party and may be accompanied by the transfer of physical ownership or ownership rights.

Reply:
Done